utils/sv_panels_props_IO_operators.py

```diff
@@ -0,0 +1,161 @@
+        if not self.filepath.endswith('.json') and not self.filepath.endswith('.JSON'):
+            self.filepath += '.json'
+        file = open(self.filepath, 'w')
+
+        export = {'node': self.target_node, 'data': []}
+
+        # Scan all node groups...
+        for tree in bpy.data.node_groups:
+
+            # ...and searching for target node
+            if tree.name == self.target_node:
+                for item in tree.Sv3DProps:
+
+                    no = item.node_name
+                    prop = item.prop_name
+                    node = tree.nodes[no]
+                    row = {'type': node.bl_idname, 'prop': prop, 'id': no}
+                    if node.label:
+                        row['label'] = node.label
+
+                    # Scalar
+                    if node.bl_idname in {"IntegerNode", "FloatNode", "SvNumberNode"}:
+                        row['value'] = getattr(node, prop, None)
+
+                    # BMesh
+                    elif node.bl_idname == "SvBmeshViewerNodeMK2":
+                        row['value'] = getattr(node, prop, None)
+                        row['material'] = node.material
+
+                    # Lists
+                    elif node.bl_idname == "SvListInputNode":
+                        row['items'] = []
+                        row['mode'] = node.mode
+                        if node.mode == 'vector':
+                            for i in range(node.v_int):
+                                row['items'].append([node.vector_list[i * 3 + j] for j in range(3)])
+                        else:
+                            for i in range(node.int_):
+                                row['items'].append(getattr(node, node.mode)[i])
+
+                    # Color
+                    elif node.bl_idname == "SvColorInputNode":
+                        row['value'] = [c for c in getattr(node, prop, None)]
+
+                    # Objects
+                    elif node.bl_idname == "SvObjectsNodeMK3":
+                        row['value'] = [pg.items()[0][1] for pg in node.object_names]
+
+                    export['data'].append(row)
+
+                # print(json.dumps(export, ensure_ascii=False, indent=2))
+                file.write(json.dumps(export, ensure_ascii=False, indent=2))
+                file.close()
+        return {'FINISHED'}
+
+    def invoke(self, context, event):
+        # event.mouse_x
+        # event.mouse_y
+        context.window_manager.fileselect_add(self)
+        return {'RUNNING_MODAL'}
+
+
+class Sv3DPanelPropsImporter(bpy.types.Operator):
+    """Import current tree settings from .JSON file"""
+    bl_idname = "node.tree_props_importer"
+    bl_label = "Import settings"
+
+    target_node = bpy.props.StringProperty(name="Target node ID")
+    filepath = bpy.props.StringProperty(subtype='FILE_PATH')
+    filter_glob = StringProperty(default="*.json", options={'HIDDEN'})
+
+    def execute(self, context):
+        file = open(self.filepath, 'r')
```

Comment:
use 
```python
open(self.filepath, 'r') as file:
    ...rest of your file writing/reading code here 
```
that way you know the file is always closed when the code exits the scope ( with-as does that automatically )